carbon/assetlist.json

```diff
@@ -60,68 +51,106 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.svg"
-      },
-      "images": [
+      }
+    },
+    {
```

Comment:
This is difficult to read. Please keep all existing assets, and add all new assets to the end of the array of assets